packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -512,7 +512,7 @@ - (int64_t)duration {
   return FVPCMTimeToMillis([[[_player currentItem] asset] duration]);
 }
 
-- (void)seekTo:(int)location completionHandler:(void (^)(BOOL))completionHandler {
+- (void)seekTo:(int64_t)location completionHandler:(void (^)(BOOL))completionHandler {
```

Comment:
We were calling this with `input.position.intValue` previously, which when I removed `intValue` to adjust for the lack of boxing turned into a warning about loss of precision passing an `NSInteger` as an `int`. But this is just used for `CMTimeMake`, which takes an `int64_t`, so it was better to change this type than to pointless cast at the call site and (in theory, but probably never in practice) truncate the value.